content/docs/command-reference/exp/apply.md

```diff
@@ -15,21 +15,25 @@ positional arguments:
 ## Description
 
 Restores an `experiment` into the workspace as long as no more Git commits have
-been made after the target experiment (`HEAD` hasn't moved). The `experiment`
-can be referenced by name or hash (see `dvc exp run` for details). This changes
-any files (code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.)
-needed to reflect the experiment conditions and results in the workspace.
+been made after the target experiment (`HEAD` hasn't moved). The experiment can
+be referenced by name or hash (see `dvc exp run` for details).
 
-⚠️ Conflicting changes in the workspace are overwritten unless `--no-force` is
-used.
+Specifically, `dvc exp apply` changes any files (code, data,
+<abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.) needed to reflect the
+experiment conditions and results in the workspace.
+
+⚠️ Current changes to the workspace are preserved except if they conflict with
```

Comment:
I think this word of caution is phrased too long. Earlier version is more to the point.

Reply:
Yeah but we wanted to add that first part. I moved it to the paragraph above to keep the warning short.